docs/python_docs/python/scripts/conf.py

```diff
@@ -69,6 +69,9 @@
 autosummary_generate = True
 numpydoc_show_class_members = False
 
+# Disable SSL verification in link check.
+tls_verify = False
```

Comment:
Does tls_verify is a GitLab Runner configuration? Shouldn't the tls_verify be disabled by default? Following the [doc](https://docs.gitlab.com/runner/configuration/advanced-configuration.html)

> tls_verify	Enable or disable TLS verification of connections to Docker daemon. Disabled by default.

Reply:
No, it's a configuration in sphinx to disable verifying the server certifications. 
ref: https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-tls_verify